<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Camera</string>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>250</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>Control</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sample</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="freezeButton">
       <property name="text">
        <string>Freeze</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sampleName"/>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="includeOverlay">
       <property name="text">
        <string>include overlay</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>250</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Overlay</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Objective Lens</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="2" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>10x</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="6">
         <widget class="QSlider" name="lensSlider">
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>4x</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>40x</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="6" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>100x</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="opacitySlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>Image</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Image scale</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="scaleSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lensSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>objectiveLensChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>overlayOpacityChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>freezeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>takeSnapshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>objectiveLensChanged(int)</slot>
  <slot>overlayOpacityChanged(int)</slot>
  <slot>takeSnapshot()</slot>
  <slot>saveImage()</slot>
 </slots>
</ui>
